zemljevid.js

```diff
@@ -332,5 +332,10 @@ let outdejtanje = setInterval(outdejtajBuse, 20000);
 var zadnjiIskaniBusId = "";
 function iskalnikBusId() {
     zadnjiIskaniBusId = prompt("Vnesi ID avtobusa", zadnjiIskaniBusId);
-    m2[zadnjiIskaniBusId].openPopup();
+
+    //Wait for busses to load
+    godusModus();
+    setTimeout(function() {
+        m2[zadnjiIskaniBusId].openPopup();
+    }, 1000);
```

Comment:
It is not 100 % that the buses will load in 1 second. Instead, since you have an async function godusModus, write 
```js
await godusModus();
m2[zadnjiIskaniBusId].openPopup();
```

To achieve that, you'll also have to make iskalnikBusId an async function.